apps/03TicTacToe.js

```diff
@@ -22,31 +28,53 @@ function printBoard() {
 }
 
 function horizontalWin() {
-    // Your code here
+    if ( (board[0][0] === board[0][1] && board[0][0] === board[0][2]) || (board[1][0] === board[1][1] && board[1][0] ===board[1][2]) || (board[2][0] === board[2][1] && board[2][2] === board[2][1]) ) 
+       {return true;}
 }
```

Comment:
Your board tests work, and are clearly well thought out.

You could just return the value of the expression:

```javascript
return ( (board[0][0] === board[0][1] && board[0][0] === board[0][2]) || (board[1][0] === board[1][1] && board[1][0] ===board[1][2]) || (board[2][0] === board[2][1] && board[2][2] === board[2][1]) );
```